client/asset/eth/eth.go

```diff
@@ -506,14 +453,18 @@ func (*ExchangeWallet) PreRedeem(req *asset.PreRedeemForm) (*asset.PreRedeem, er
 
 // coin implements the asset.Coin interface for ETH
 type coin struct {
-	id srveth.AmountCoinID
+	id dex.Bytes
```

Comment:
I'll leave this part for now, the previous agreement was to just send the transaction hash, but I'm thinking it would be better to send the entire `TxCoinID` including the index. I commented over here:
https://github.com/decred/dcrdex/issues/1154#issuecomment-979251509